Initial UI

shubhscoder opened the pull request:
## Ready for merge?
**NO**

#### What does this PR do?
The UI for statistics dashboard

#### How can a reviewer manually see the effects of these changes?
I will be adding screenshots, once controllers are setup

#### Screenshots (if appropriate)

#### Todo:
- [ ] Tests
- [ ] Documentation
- [ ] Stakeholder approval

#### Requires Database Migrations?
NO

#### Includes new or updated dependencies?
YES

thatandromeda commented:
I think I may have fixed this test failure in a different branch -- check to see if you're on the latest version of dev and rebase if not.

shubhscoder commented on app/assets/stylesheets/application.css.scss:
```diff
@@ -46,6 +46,7 @@
 @import 'blog/show';
 @import 'counter_notices/show';
 @import 'token_urls/index';
+@import 'statistics/statistics_notices';
```
:+1:

shubhscoder commented on app/controllers/statistics_controller.rb:
```diff
@@ -0,0 +1,67 @@
+	def sidebar_items
+		[
+		 "notices",
+		 "infringing url's",
```
Happened by mistake. Done . Thank you!

shubhscoder commented on app/controllers/statistics_controller.rb:
```diff
@@ -0,0 +1,67 @@
+	before_action :set_commons
+	def statistics_notices
+		@notices_count = Notice.get_approximate_count
+		@average_count = @notices_count / ([1, Entity.count].max)
```
Yes, I had caught that. But taking into consideration, the number of notices and URL's I thought its fine to approximate the values also notices_count is approximate. Thank you :+1:

shubhscoder commented:
> To change:
> 
> * [ ]  The stats stylesheet applies to elements found sitewide (e.g. headers, links, `body` (!)) but is only meant to affect the stats page. Please make it more specific (e.g. selectors like `#stats-dashboard h4`), or only load it on stats pages.

I have refactored the entire stylesheet to increase readability and maintainability in #572. This also takes care of the above change.
> * [ ]  Change "url's" or "URL's" to "URLs" wherever they appear

Done. Thank you!
> * [ ]  If the dashboard template came from elsewhere, provide a link to the source

Actually I have'nt referred one template, I used small small pieces from many, I will try to find them, but the code is changed drastically, still I will try to find them again.

> * [ ]  Add tests to the empty test coverage file (some good tests: that the routes resolve with 200 OK; that your public controller methods yield the data you expect -- no need to test private ones)

I have added some simple tests, because there is nothing really complex in the controllers. If you want any specific tests, then I can implement them

thatandromeda commented:
As you've merged this into #572, I'm going to close this PR. I'll check for the todo items in 572.
